package.json

```diff
@@ -15,8 +15,10 @@
     "actsAs": [
       "handler"
     ],
+    "type": "handler",
```

Comment:
`actsAs` should replace `type`, which is deprecated. What led you to add it here?

Reply:
Confusion.  For some reason, I read your comment about "type" being displaced in favor of "actAs" perfectly in reverse, and thought actAs was the outdated syntax, leading me to add "type."  And then I just forgot to remove it.  I'll take it out.